### PR TITLE
Add BOB token

### PR DESCRIPTION
**Pull request by aalan3**
Brief comments on the purpose of your changes:

### Pricing checks:
* [x] `coin_id` represents the ID of the coin on coinpaprika.com

### CODE REVIEW

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -316,6 +316,8 @@ VALUES
     ("ape-apecoin", "bnb" ,"APE" ,"0x0b079b33b6e72311c6be245f9f660cc385029fc3",18),
     ("metis-metis-token","bnb" ,"Metis" ,"0xe552fb52a4f19e44ef5a967632dbc320b0820639" ,18),
     ("xen-xen-crypto", "bnb", "XEN", "0x2AB0e9e4eE70FFf1fB9D67031E44F6410170d00e", 18),
+    ("bob-bob", "ethereum", "bnb", "0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b", 18),
+
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:320]: I am not sure about this line, but "bnb" is at the column `symbol`, maybe this is the targeted change:
```suggestion
    ("bob-bob", "ethereum", "BOB", "0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b", 18),

```
However this change will make this line the same as line 1616:
https://github.com/duneanalytics/spellbook/pull/2373/files#diff-0350e0e6bf4dcce94ae74401c31cb3fe946eaaef8f2b2faa63ad2f4c32d6dbb7R1616

Comment (Hosuke) [reply models/prices/prices_tokens.sql:320]: Or maybe this:
```suggestion
    ("bob-bob", "bnb", "BOB", "0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b", 18),

```
as it is also the addresses on bsc: https://bscscan.com/address/0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b

**Review by Hosuke — APPROVED** [review]
This PR adds BOB token to prices.usd table, for [ethereum, optimism, polygon] (and bnb possibly) chains:
https://api.coinpaprika.com/v1/coins/bob-bob
- `is_active: true` ✅
- correct addresses on different chains ✅ 

Token addresses on different chains:
- ethereum: https://etherscan.io/token/0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b
- bnb: https://bscscan.com/token/0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b
- optimism: https://optimistic.etherscan.io/token/0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b
- polygon: https://polygonscan.com/token/0xB0B195aEFA3650A6908f15CdaC7D92F8a5791B0B

I will fix the line of bob-bob for bnb chain.

File: models/prices/prices_tokens.sql (hunk 1 above)
Comment (aalan3) [reply models/prices/prices_tokens.sql]: Good catch thank you!

**Comment by dot2dotseurat** [comment]
@Hosuke so the address is the same for all platforms?

**Review by dot2dotseurat — APPROVED** [review]

**Comment by Hosuke** [comment]
> @Hosuke so the address is the same for all platforms? Looks like the api only have that polygon contract so I can't update the automation to make this pass but I'll note this because this makes the error message confusing.

Yes, but the test information still helps(a lot), currently for prices.usd address updates we may still need to check manually.

**Pull request merged**